alibi_detect/cd/__init__.py

```diff
@@ -1,8 +1,10 @@
 from .chisquare import ChiSquareDrift
 from .classifier import ClassifierDrift
 from .ks import KSDrift
+from .learnt_kernel import LearntKernelDrift
```

Comment:
It becomes clear we are not a US-based drift detection library here. :)

Reply:
Haha I actually think I need to change every instance of 'learnt' to 'learned'. Turns out 'learnt' is standard outside US for the past participle but as an adjective should always be 'learned'.